helmholtz-code/piCode/uart_test.py

```diff
@@ -10,9 +11,9 @@
     # detect the operating system, if windows, replace forward slashes with backslashes
     # code will work for macOS, linux, and windows
     pathstr = "../TestData/ThinSat_MagField_Vector.csv"
-    """if platform.system() == 'Windows':
+    if platform.system() == 'Windows':
```

Comment:
Instead of this if statement, consider using [`os.path.join`](https://docs.python.org/3.3/library/os.path.html#os.path.join). Also, I think Windows would require a double backslash, not a single backslash.

e.g. `str(os.path.join("..", "TestData", "ThinSat_MagField_Vector.csv"))`